test/test_other.py

```diff
@@ -10950,6 +10950,26 @@ def check_func_info(filename, address, func):
     # The name section will not show bar, as it's inlined into main
     check_func_info('test_dwarf.wasm', unreachable_addr, '__original_main')
 
+    # 2. Test symbol map
```

Comment:
Maybe make this a separate test?

Reply:
Yes, that (function-only vs src location granularity) was my rationale for keeping this together with the name section tests. The test assertions are basically duplicated (DAMP-style) but it seemed best to keep them togeher. But if you feel strongly we can split both of these tests up.